src/Corax/Utils/GrowablePooledBuffer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
+using Voron;
+
+namespace Corax.Utils;
+
+public struct GrowablePooledBuffer<T> : IDisposable
+{
```

Comment:
Should be removed if not used. And if we are going to keep it (because it is properly tested) it must go in `Sparrow.Memory`

Reply:
done